src/controllers/ioSignController.ts

```diff
@@ -22,18 +22,36 @@ import IoSignService from "src/services/ioSignService";
 import { pipe } from "fp-ts/lib/function";
 import { FiscalCode, NonEmptyString } from "@pagopa/ts-commons/lib/strings";
 import { errorsToReadableMessages } from "@pagopa/ts-commons/lib/reporters";
-import { QtspClausesMetadataDetailView } from "generated/io-sign/QtspClausesMetadataDetailView";
+import { Errors } from "io-ts";
+import { Id } from "../../generated/io-sign/Id";
+import { QtspClausesMetadataDetailView } from "../../generated/io-sign/QtspClausesMetadataDetailView";
+import { SignatureDetailView } from "../../generated/io-sign/SignatureDetailView";
+import { SignatureRequestDetailView } from "../../generated/io-sign/SignatureRequestDetailView";
 import { SignerDetailView } from "../../generated/io-sign-api/SignerDetailView";
-
 import { FilledDocumentDetailView } from "../../generated/io-sign/FilledDocumentDetailView";
 
 import { CreateFilledDocument } from "../../generated/io-sign/CreateFilledDocument";
+import { CreateSignatureBody } from "../../generated/io-sign/CreateSignatureBody";
 
 import { withUserFromRequest } from "../types/user";
 import ProfileService from "../services/profileService";
 
 import { profileWithEmailValidatedOrError } from "../utils/profile";
 
+const responseErrorValidation = (errs: Errors) =>
+  ResponseErrorValidation(
+    "Bad request",
+    errorsToReadableMessages(errs).join(" / ")
+  );
+
+const responseErrorInternal = (reason: string) => (e: Error) =>
+  ResponseErrorInternal(`${reason} | ${e.message}`);
+
+const toErrorRetrievingTheSignerId = (e: Error) =>
```

Comment:
I'd just use `responseErrorInternal` inline and remove this method.

Reply:
`responseErrorInternal` is used in many places with the same reason. Do you think it's better to disable the lint warning?

Reply:
yep